lldb/include/lldb/Symbol/SymbolContext.h

```diff
@@ -201,8 +201,8 @@ class SymbolContext {
   bool GetAddressRange(uint32_t scope, uint32_t range_idx,
                        bool use_inline_block_range, AddressRange &range) const;
 
-  bool GetAddressRangeFromHereToEndLine(uint32_t end_line, AddressRange &range,
-                                        Status &error);
+  llvm::Expected<bool> GetAddressRangeFromHereToEndLine(uint32_t end_line,
```

Comment:
Is the `bool` value actually important? If not, this could be an `llvm::Error` to simplify the interface.